CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# How To Contribute
+
+Thanks for reading our contribution guidelines.
+
+The following guidelines for contribution should be followed if you want to submit a pull request.
+
+## How To Prepare
+
+* You need a [GitHub account](https://github.com/signup/free)
+* Submit an [issue ticket](https://github.com/thomasgriffin/TGM-Plugin-Activation/issues) for your issue if there is not one yet.
+    * Describe the issue and include steps to reproduce if it's a bug.
+    * Include relevant version numbers.
+    * Additional screenshots or videos are often helpful.
```

Comment:
* If the issue is a bug, add any relevant JS/PHP error messages. For PHP error message, a backtrace is preferred. If you don't know how to get one, please [follow these instructions](https://gist.github.com/jrfnl/5925642).